modules/claims_api/config/routes.rb

```diff
@@ -59,6 +59,12 @@
     end
   end
 
+  namespace :v2, defaults: { format: 'json' } do
+    mount OkComputer::Engine, at: '/healthcheck'
+
+    post '/veteran-id', to: 'veteran_identifier#create'
```

Comment:
linking back to this discussion:
https://github.com/department-of-veterans-affairs/vets-api/pull/6845#discussion_r632018043

bit of a nitpick probably, but does it make more sense to call this `search` or `find` instead of `create` ?

Reply:
Alright, you wore me down.
Custom method it is.  😂 
[Changed Here](https://github.com/department-of-veterans-affairs/vets-api/pull/6845/commits/b133d9f2bd7591480bc9d243e80e7cd1a4c3ac75)

Reply:
hahahah but :(
im sorry :( :( :( 
but thank you ❤️